src/file.cpp

```diff
@@ -2192,57 +2116,6 @@ typedef struct _FILE_ALLOCATED_RANGE_BUFFER {
 #endif
 	}
 
-	boost::int64_t file::sparse_end(boost::int64_t start) const
```

Comment:
This function was used in 1.0.x to skip sparse sections of files when checking. in 1.1 the checking mechanism was moved out of the disk thread into the torrent, which now just ask to hash pieces. I think it would still be useful to skip pieces that fall into sparse sections of files. it can be a lot faster. I suppose I left this function dangling when making that transition. One possibility to make use of this would be to have the hash disk job also return the offset to the next valid file data, or something along those lines.